src/model/LabyrinthMatrix.java

```diff
@@ -1,73 +0,0 @@
-package model;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthMatrix extends LabyrinthImpl {
-
-	private int matrix[][];
-	private int  freeValue, wallValue;
-
-	public LabyrinthMatrix(int rows, int columns) {
-		this.rows = rows;
-		this.columns = columns;
-
-		matrix = new int[rows][columns];
-	}
-
-	public LabyrinthMatrix() {
-	}
-
-	public void loadFromFile(String path) {
-		Scanner sc;
-		try {
-			sc = new Scanner(new File(path));
-			rows = sc.nextInt();
-			columns = sc.nextInt();
-			matrix = new int[rows][columns];
-			for (int i = 0; i < rows; i++)
```

Comment:
Just as a note, ++something is usually preferred to something++ because the postfix creates one extra copy for the object being incremented. For a mere "int" that's not such a big deal, but it's a good thing if you get that "in your blood" :).